packages/commonwealth/server/routes/searchComments.ts

```diff
@@ -53,7 +53,7 @@ const searchComments = async (
   SELECT
       "Threads".title,
       "Comments".text,
-      "Comments".root_id as proposalId,
+      "Comments".thread_id as proposalId,
```

Comment:
Do we need to modify this call at all, to deal with the change in format?

Reply:
It is not needed to modify to deal with a change in format, but due to this format it can (and I will) be modified to make the query more efficient.